owncloudApp/src/main/res/values/strings.xml

```diff
@@ -717,5 +717,7 @@
     <string name="release_notes_3.0_subtitle5">With the redesign of the sync engine, now several downloads and uploads can run at the same time</string>
     <string name="release_notes_3.0_title6">Browser login improved</string>
     <string name="release_notes_3.0_subtitle6">Only typing the URL, browser will be triggered</string>
+    <string name="release_notes_3.0.1_title1">Database migration fixed</string>
```

Comment:
Users probably don't care about the technical issue, but the `Fix crash when upgrading from 2.18 ` or somehting like that, don't u think?